src/custom/components/Popups/WarningPopup.tsx

```diff
@@ -0,0 +1,24 @@
+import { useContext } from 'react'
+import styled, { ThemeContext } from 'styled-components/macro'
+
+import { ThemedText } from 'theme'
+import { AutoColumn } from 'components/Column'
+import { AutoRow } from 'components/Row'
+
+const RowNoFlex = styled(AutoRow)`
+  flex-wrap: nowrap;
```

Comment:
I guess it will need some help with CSS, at least for the color of the text. 

If it's warning, should we add some warning icon? 

This warning is meant to be shown only in developing time, so no really an issue to style it amazingly, but nice to add the warning icon anyways.